src/java/org/apache/cassandra/io/util/PathUtils.java

```diff
@@ -334,6 +336,52 @@ private static void deleteRecursive(Path path, RateLimiter rateLimiter, Consumer
         delete(path, rateLimiter);
     }
 
+    /**
+     * Recursively delete the content of the directory, but not the directory itself.
+     */
+    public static void deleteContent(Path path)
+    {
+        deleteContent(new File(path));
+    }
+
+    /**
+     * Recursively delete the content of the directory, but not the directory itself.
+     */
+    public static void deleteContent(File dir)
+    {
+        if (!dir.exists())
+            return;
+
+        for (File child : dir.tryList())
+            deleteRecursive(child.toPath());
+    }
```

Comment:
This is the only method with argument of `File`, so I don't think it fits `PathUtils`. I suggest to remove it from here.

Reply:
Yes, in fact uses of this method can be replaced with the one taking `Path`, so it's not needed at all.